Lib/test/support/strace_helper.py

```diff
@@ -0,0 +1,173 @@
+import re
+import sys
+import textwrap
+import unittest
+from dataclasses import dataclass
+from test import support
+from test.support.script_helper import run_python_until_end
+from typing import Dict, List
```

Comment:
FYI - modern typing does not need these two.  use dict[] and list[] directly.  (but read below, we aren't ready to accept non-trivial annotations in test.support)